SimPEG/Examples/DC_PseudoSection_Simulation.py

```diff
@@ -0,0 +1,220 @@
+from SimPEG import *
+# import simpegDCIP as DC
+from SimPEG import DCIP as DC
+import scipy.interpolate as interpolation
+import matplotlib.pyplot as plt
+import time
+import re
+
+def run(loc=np.c_[[-50.,0.,-50.],[50.,0.,-50.]], sig=np.r_[1e-2,1e-1,1e-3], radi=np.r_[25.,25.], param = np.r_[30.,30.,5], stype = 'dpdp',  plotIt=True):
+    """
+
+        DC Forward Simulation
+        =====================
+
+        Forward model conductive spheres in a half-space and plot a pseudo-section
+
+        Created on Mon Feb 01 19:28:06 2016
```

Comment:
I just want to time the operations since using an iterative solvers, and can get long.
Is there a better functions?